rebar.lock

```diff
@@ -1,17 +1,17 @@
 {"1.2.0",
 [{<<"aten">>,{pkg,<<"aten">>,<<"0.5.8">>},1},
  {<<"gen_batch_server">>,{pkg,<<"gen_batch_server">>,<<"0.8.8">>},1},
- {<<"ra">>,{pkg,<<"ra">>,<<"2.1.0">>},0},
+ {<<"ra">>,{pkg,<<"ra">>,<<"2.4.0">>},0},
```

Comment:
2.4.1 is released now as well (it bumps seshat). I can bump from 2.4.0->2.4.1 after this is merged since the commit that bumps to 2.4.0 has details on what in 2.4.0 is necessary for this feature

Reply:
Ok